tendermint/src/lite/verifier.rs

```diff
@@ -784,20 +782,18 @@ mod tests {
         assert_single_err(ts, vac, err.into());
     }
 
-    #[tokio::test]
-    async fn test_verify_bisection_1_val() {
```

Comment:
Ok, answers my question I guess :-D 
https://github.com/informalsystems/tendermint-rs/pull/338/files#r441736094

Reply:
Also, I should better read the introductory comments 🙈 😆 

> Controversially this also removes the dependency on tokio as it was mostly used for async test execution, which was achieved by making extensive use of futures::executor::block_on